examples/parcel/.compiledcssrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is `.compiledcssrc.json` preferred over `compiledcss.js`?

Reply:
Yes!